### DISCUSSION TITLE
无法推送

### DISCUSSION THREAD

**Discussion by LOVEYIKANUOSI** [VickScarlet/lifeRestart#327] [Q&A]
萌新加尝试加了一个天赋在本地正常运行但是在推送的时候提示：
正在推送 master
Remote: Permission to VickScarlet/lifeRestart.git denied to LOVEYIKANUOSI.
未能推送到远程存储库。请参见输出窗口了解更多详细信息。
发布到远程存储库时遇到错误: Git failed with a fatal error.
Git failed with a fatal error.
unable to access 'https://github.com/VickScarlet/lifeRestart.git/': The requested URL returned error: 403

这是因为我本地git的原因还是因为我没有上传的权限

**Comment by VickScarlet** [comment]
如果是自己改着玩，那可以 fork 仓库，然后推送到自己 fork 的仓库，如果是想提到主仓库，可以推到自己 fork 的仓库后提一个 pull request

**Reply by LOVEYIKANUOSI** [reply]
大佬我没有用code我用的是vs2019我没不知道“pull request"这个是什么操作
![image](https://user-images.githubusercontent.com/46013924/134343753-6326133b-bd62-4654-a166-fe70feb868e0.png)

**Reply by VickScarlet** [reply]
> 大佬我没有用code我用的是vs2019我没不知道“pull request"这个是什么操作
> ![image](https://user-images.githubusercontent.com/46013924/134343753-6326133b-bd62-4654-a166-fe70feb868e0.png)

@LOVEYIKANUOSI  看看 GitHub 的帮助文档 [Creating a pull request from a fork](https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork)
